packages/connectivity/connectivity/example/lib/main.dart

```diff
@@ -103,7 +103,7 @@ class _MyHomePageState extends State<MyHomePage> {
         String wifiName, wifiBSSID, wifiIP;
 
         try {
-          if (Platform.isIOS) {
+          if (!kIsWeb && Platform.isIOS) {
```

Comment:
Doesn't `Platform.isIOS` already imply "not web"?

Reply:
It does, however the boolean check is required to short-circuit the if and prevent Platform.isIOS from actually being called; it breaks on web (for now)